modules/SofaValidation/src/SofaValidation/EvalPointsDistance.h

```diff
@@ -120,12 +120,12 @@ class EvalPointsDistance: public virtual sofa::core::objectmodel::BaseObject
     {
         std::string object1 = arg->getAttribute("object1","@./");
         std::string object2 = arg->getAttribute("object2","@./");
-        if (!LinkMState::CheckPath(object1, context)) {
+        if (!sofa::core::PathResolver::CheckPath(context, LinkMState::DestType::GetClass(), object1)) {
```

Comment:
Still issues with PathResolver here and there

Reply:
Actually... they are deleted !!! So it was perfectly in line with the topic :)